Dockerfile-node

```diff
@@ -1,18 +1,16 @@
-FROM node:6.2
+FROM node:8.5
 LABEL maintainer "ODL DevOps <mitx-devops@mit.edu>"
 
-RUN apt-get update && apt-get install libelf1
-
-RUN adduser --disabled-password --gecos "" mitodl
+RUN apt-get update && apt-get install libelf1 -y
 
 COPY package.json /src/
 
 COPY scripts /src/scripts
 
 RUN node /src/scripts/install_yarn.js
 
-RUN mkdir -p /home/mitodl/.cache/yarn
+RUN mkdir -p /home/node/.cache/yarn
 
-RUN chown mitodl:mitodl /home/mitodl/.cache/yarn
+RUN chown node:node /home/node/.cache/yarn
 
-USER mitodl
+USER node
```

Comment:
Any particular reason for change of user?

Reply:
Yes, in the updated Docker container there is an existing `node` user which has the 1000 uid, which is the default for non-system users. The uid must match between the users for the docker volume sharing to let the desktop user edit files from the docker user and vice versa.